Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Properties.cs

```diff
@@ -26,9 +26,21 @@ public partial class ImageCropper
         public double MinSelectedLength { get; set; } = 40;
 
         /// <summary>
-        /// Gets the current cropped region.
+        /// Gets or sets the current cropped region.
         /// </summary>
-        public Rect CroppedRegion => _currentCroppedRect;
+        public Rect CroppedRegion
+        {
+            get
+            {
+                return _currentCroppedRect;
+            }
+
+            set
+            {
+                _currentCroppedRect = value;
```

Comment:
We need to check the value of `_currentCroppedRect` to ensure it is valid (related to `_restrictedCropRect` and `MinCropSize`), otherwise the saved image will be incorrect.

Reply:
In addition, we have to consider `AspectRatio`(`KeepAspectRatio` and `UsedAspectRatio` are used in the code).

Reply:
Good point. What do you suggest to do when the provided rect doesn't satisfy the requisites? 

Thinking out loud: We could modify it so it respects the minimum size and aspect ratio, but it would be hard for the user of the control to predict what the resulting rect will be. Maybe we should just drop it when the value is not valid? But since this is a setter there will be no failure indication, unless we throw and exception which might be too aggressive. 

What do you think about discarding the setter and replacing it with
```csharp
bool TrySetCroppedRegion(Rect r)
```
returning true if the rect satisfies the requirements, and false if the rect is discarded?

Reply:
I agree. It's a great idea！